Source/Atomic/UI/UIWidget.h

```diff
@@ -250,10 +258,18 @@ class UIWidget : public Object, public tb::TBWidgetDelegate
     bool GetCaptured();
 
     void SetCapturing(bool capturing);
-    bool GetCapturing() { return widget_->GetCapturing(); }
+    bool GetCapturing();
 
     void InvokeShortcut(const String& shortcut);
 
+    bool GetShorten();
```

Comment:
GetShorten returns true if widget has a string and that string has its length > widget rect, so that string has ...(three dots) at the end
![](http://puu.sh/lOoJ9/07a7461b1a.png)